dtm0/it/all2all/README.md

```diff
@@ -43,3 +43,41 @@ it needs to be patched by the hare.patch in current directory
 to make it able to generate clusted configuration that includes
 DTM0 service. After that it should be built and installed into
 the system (see [README.md](https://github.com/Seagate/cortx-hare/blob/main/README.md) in the root of Hare repo).
+
+## Troubleshooting
+### 1. Test fails for starting motr processes
+If test fails at the stage where it start motr processes, please check if /etc/modprobe.d/lnet.conf has same interface as 
```

Comment:
s/where it start motr processes/where it starts Motr processes

Reply:
Sure will correct